front/pages/w/[wId]/builder/data-sources/[name]/index.tsx

```diff
@@ -63,6 +63,8 @@ const {
   NANGO_CONFLUENCE_CONNECTOR_ID = "",
   NANGO_GOOGLE_DRIVE_CONNECTOR_ID = "",
   NANGO_INTERCOM_CONNECTOR_ID = "",
+  NANGO_MICROSOFT_SHAREPOINT_CONNECTOR_ID = "",
```

Comment:
Do we need 2 Nango connections for Teams and Sharepoints givent that they share the same API?

Should this be a Microsoft Suite connector? or should they be separate?

Reply:
Can an App be shared among 2 connections from the same user?

Reply:
Updated the connectors to use the same app